cas9-AA-subst.py

```diff
@@ -0,0 +1,51 @@
+#///////////////////////**READ ME FIRST**//////////////////////////////////
+# Point of this script is to substitute amino acids into a given pose 
+#  protein structure in pyrosetta.
+# Please change dropbox folder location accordingly + make sure that the 
+#  script's current location is the root of PyRosetta (need to import stuff)
+#//////////////////////////////////////////////////////////////////////////
+## Raw file date modified: July 15th/2014
+
+## importing modules required for script
+import os
+from rosetta import *
+## difference between resfiles and mutate_residue is that the former takes into account 
+##  rotamers (aka conformers) whereas the latter doesn't (for our case mutate_residue is sufficient)
+##  refer to link for further documentation and/or info:
+##  http://graylab.jhu.edu/pyrosetta/downloads/documentation/Workshop6_PyRosetta_Packing_Design.pdf
+from toolbox import generate_resfile_from_pdb # generate mutations using resfiles
+from toolbox import mutate_residue # generate mutations using mutate_residue  
+
+## changing directory to where PDB's are located (aka where PDB files are located )
+os.chdir("~\Dropbox\Waterloo-iGEM-2015") #alter to your specific dropbox path
+os.chdir("\Math Modelling\cas9_modification") ##where the WT cas9 should be located???
+## not sure if completely correct???? add changes if not.
+
+## initializing rosetta:
+rosetta.init()
+
+# import cleaning module for PDB to be usable
+from toolbox import cleanATOM
+
+cleanATOM("\4UN3.pdb") # cleaned PDB file to use for analysis
+var_pose = pose_from_pdb("\4UN3.pdb") # initial pose created from clean pdb
+
+#inputted residue number of interest
+Num = raw_input("enter residue number:\n")
+
+for i in range(0, 20):
```

Comment:
So I think you could `from cas9_mutants import klienstiver_mutants` at the beginning and then this section should be approximately (note I am typing this in a text box not editor, may not compile):
```
for cas9 in klienstiver_mutants:
    for mutation in cas9["mutations"]
        var_pose = pose_from_pdb("4UN3." + mutation["aa_mut"] + ".clean.pdb")
        mutate_residue(var_pose, mutation["aa_idx"], mutation["aa_mut"])
```
which would then apply all the Kleinstiver mutations!

I am confused about how more than one mutation is applied at a time here... do a bunch of **pose_from_pdb** files get combined into a single altered pdb at the end?